.ci_support/migrations/cuda110.yaml

```diff
@@ -0,0 +1,42 @@
+migrator_ts: 1601612527
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+  override_cbc_keys:
+    - cuda_compiler_stub
+
+cuda_compiler_version:
+  - None
+  - 9.2                        # [linux64]
+  - 10.0                       # [linux64]
+  - 10.1                       # [linux64]
+  - 10.2                       # [linux64]
+  - 11.0                       # [linux64]
+
+# cdt_name:  # [os.environ.get("BUILD_PLATFORM", "").startswith("linux") or (os.environ.get("CONFIG_VERSION", "1") == "1" and linux)]
```

Comment:
Should this block still be present?

Reply:
This inherited from https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/884